fr/integrations/plugin-packs/procedures/network-switchs-symbol-wing-snmp.md

```diff
@@ -0,0 +1,178 @@
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prérequis
+
+Pas de pré-requis particuliers
+
+### Flux réseau
+
+La communication doit être possible sur le port UDP 161 depuis le collecteur
+Centreon vers le serveur Linux supervisé.
+
+## Installation
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Online License-->
+
+1. Installer le Plugin Centreon sur tous les collecteurs Centreon devant superviser des resources *Symbol WiNG*:
+
+```bash
+yum install centreon-plugin-Network-Symbol-Wing-Snmp
+```
+
+2. Sur l'interface Web de Centreon, installer le Plugin Pack *Symbol WiNG switch SNMP* depuis la page **Configuration > Packs de plugins**
+
+<!--Offline IMP License-->
+
+1. Installer le Plugin Centreon sur tous les collecteurs Centreon devant superviser des resources *Symbol WiNG*:
+
+```bash
+yum install centreon-plugin-Network-Symbol-Wing-Snmp
+```
+
+2. Sur le serveur Central Centreon, installer le RPM du Pack *Symbol WiNG switch SNMP*:
+
+ ```bash
+yum install centreon-pack-network-switchs-symbol-wing-snmp
+```
+
+3. Sur l'interface Web de Centreon, installer le Plugin Pack *Symbol WiNG switch SNMP* depuis la page **Configuration > Packs de plugins**
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Configuration
+
+### Hôte
+
+* Ajoutez un Hôte à Centreon depuis la page **Configuration > Hôtes**.
+* Complétez les champs "Nom","Alias" & "IP Address / DNS" correspondant à votre serveur *Symbol WiNG*
+* Appliquez le Modèle d'Hôte **network-switchs-symbol-wing-snmp-custom** 
+
+Si vous utilisez SNMP en version 3, vous devez configurer les paramètres
+spécifiques associés via la macro SNMPEXTRAOPTIONS.
+
+| Mandatory | Name             | Description                                              |
+|:----------|:-----------------|:---------------------------------------------------------|
+|           | SNMPEXTRAOPTIONS | (Default: 'Configure your own SNMPv3 credentials combo') |
+
+## Comment puis-je tester le Plugin et que signifient les options des commandes ? 
+
+Une fois le Plugin installé, vous pouvez tester celui-ci directement en ligne 
+de commande depuis votre collecteur Centreon en vous connectant avec 
+l'utilisateur *centreon-engine*:
+
+```bash
+/usr/lib/centreon/plugins//centreon_symbol_wing.pl \
+    --plugin=network::symbol::wing::snmp::plugin \
+    --mode=systems \
+    --hostname='10.0.0.1' \
+    --snmp-community='my-snmp-community' \
+    --snmp-version='2c' \
+    --filter-name='' \
+    --warning-cpu-utilization-1m='' \
+    --critical-cpu-utilization-1m='' \
+    --warning-cpu-utilization-5m='80' \
+    --critical-cpu-utilization-5m='90' \
+    --warning-cpu-utilization-15m='' \
+    --critical-cpu-utilization-15m='' \
+    --warning-memory-usage='' \
+    --critical-memory-usage='' \
+    --warning-memory-usage-free='' \
+    --critical-memory-usage-free='' \
+    --warning-memory-usage-prct='' \
+    --critical-memory-usage-prct='' \
+    --warning-devices-total='' \
+    --critical-devices-total='' \
+    --use-new-perfdata 
```

Comment:
--use-new-perfdata is useless in the _systems_ mode as the new format is forced (see here https://github.com/centreon/centreon-plugins/blob/a8b6ea80d78b49a13aa9245a5397f9179ef8a20f/network/symbol/wing/snmp/mode/systems.pm#L141)
```suggestion

```